### PR TITLE
Responsive event grid

### LINKED ISSUE #226: Mejorar la vista de la grilla en eventos de en tamaño 1024
**Explanations**

when entering the laptop view of 1024 only one element is seen in the events grid, it should show two as in products / Al  ingresar a la vista de laptop de 1024 se ve solo un elemento en la grilla de eventos, debería mostrarse dos como en productos. 

**Actual/current** 

image

**Expected result**

image

- [ ] Cambiar la vista de la grilla a dos elementos / Change the grid view to two elements

### PR DESCRIPTION
**Pull request by Eghinner**
### Description

When the dimensions is 1600px or less can see 2 columns

fixes #226 

#### Screenshots
chrome_Ile2fgzkDH

#### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### How to test it

1. Go to 'events.'
2. Change dimensions

### CODE REVIEW

File: styles/Events.module.scss [hunk 1]
```diff
@@ -76,32 +83,36 @@
...
-  margin: auto;
-  text-align: left;
-  width: 300px;
+  margin-top: 1.25rem;
+  max-width: 90%;
 
   h2 {
     -webkit-box-orient: vertical;
+    /* stylelint-disable-next-line value-no-vendor-prefix */
+    display: -webkit-box;
```
Comment (Israel-Laguan) [inline styles/Events.module.scss:122]: Why is this needed?

**Comment by angelik0828** [comment]
## ✔️ The change looks pretty good just a comment 
- When you are on bigger screens you should see more than two items 

image

**Current**

image

**Expected result in resolutions of more than 2500 px**

image

File: styles/Events.module.scss (hunk 1 above)
Comment (Eghinner) [reply styles/Events.module.scss:122]: becuse ~~`-moz-box-flex`~~ `-webkit-box` is diferent than `box-flex`
https://developer.mozilla.org/es/docs/Web/CSS/box-flex
stylelint pretends not to need prefix, but it this case, yes

Comment (Israel-Laguan) [reply styles/Events.module.scss:122]: I was checking in Mozilla firefox but this style is not present, seems like SASS removes that style while compiling

image

Comment (Israel-Laguan) [reply styles/Events.module.scss:122]: In the docs it mention those are not handled internally by autoprefixer
https://github.com/postcss/autoprefixer
https://github.com/postcss/autoprefixer#how-to-work-with-legacy--webkit--only-code

Comment (Israel-Laguan) [reply styles/Events.module.scss:122]: Please confirm

Comment (Eghinner) [reply styles/Events.module.scss:122]: Seems that Mozilla firefox compiles correctly `-webkit-box`

**Comment by Eghinner** [comment]
@angelik0828 can you review?

**Review by angelik0828 — APPROVED** [review]
## ✔️ The responsive is working pretty well, good job!

image
image
